# Bug 1753402 - Disable untagged union in Gecko profiler bindings.
diff --git a/tools/profiler/rust-api/build.rs b/tools/profiler/rust-api/build.rs
--- a/tools/profiler/rust-api/build.rs
+++ b/tools/profiler/rust-api/build.rs
@@ -56,16 +56,17 @@ fn add_include(name: &str) -> String {
     println!("cargo:rerun-if-changed={}", file_path);
     file_path
 }
 
 fn generate_bindings() {
     let mut builder = Builder::default()
         .enable_cxx_namespaces()
         .with_codegen_config(CodegenConfig::TYPES | CodegenConfig::VARS | CodegenConfig::FUNCTIONS)
+        .disable_untagged_union()
         .size_t_is_usize(true);
 
     for dir in SEARCH_PATHS.iter() {
         builder = builder.clang_arg("-I").clang_arg(dir.to_str().unwrap());
     }
 
     builder = builder
         .clang_arg("-include")




